### ISSUE TITLE
[Suggestion?]Outlogs

### ISSUE THREAD

**Issue by MRonlin** [tsunamayo/Starship-EVO#2236]
Not sure if this is practical, but having the game store all previous outlogs.

**Comment by tsunamayo** [comment]
Yes I want to do that - keep a backup of the previous log. Thanks
